Create navy missions intro.txt (#5434)
endless-sky/endless-sky: data/human/navy missions intro.txt

		ship "Clipper (Heavy)" "Black Fang"
		dialog `The <npc> has been eliminated. You can claim the bounty payment by returning to <destination>.`

	on complete
		payment 175000
		dialog `Upon landing, the commander greets you, seemingly surprised by your return. "The Republic needs more captains like you. Hopefully this payment of <payment> will be enough to convince you to help us out the next time." You nod. "Captain <last>, correct?" he says, and barely gives you enough time to agree before he continues speaking. "Right. I'm Commander Kent, in case I didn't introduce myself. I'll be keeping an eye out for you in case we ever need more assistance." `

mission "Republic Bounty 2"
	name "Republic Bounty Hunting"
	description `Kill a known criminal aboard the <npc>, which was last spotted in the Alnasl system.`
	source
		government "Republic"
		attributes "dirt belt"
	to offer
		has "Republic Bounty 1: done"
		not "chosen sides"
		random < 70

	on offer
		conversation
			`Before you can decide where you want to go in the spaceport, you are flagged down by a Republic officer. This time, they're a dour woman with black hair in a tight bun. "Captain <last>?" she asks.`

> 			choice

Reviewer: This was our take on the same feedback. She doesn't ask, she makes a declaration of "I already know who you are, you will do this." with a forceful tone.

https://github.com/kikotheexile/Endless-Sky-Civil-War/blob/5b110944e5ad8627e62b7fc1f2152b79ed4550af/data/mainplot/story%20republic%20navy.txt#L68-L69

			`You are dead. Not big surprise.`
				die
			label yes
			`	"I thought I recognized you," he remarks with a hard look. "You wouldn't happen to know the name 'Lucas Hadley,' would you?"`
			branch uhoh
				has "bad look"
			`	"Maybe. Why do you ask?"`
				goto next
			label uhoh
			`Though you don't display it outwardly, you realize that must be the dockworker you almost attacked on Lodestone. Better choose your words carefully, or things might get ugly.`
			choice
			`	"No...why do you ask? And what does he want?"`
				goto oops
			`	"I know I acted out of line on Lodestone - I promise it won't happen again."`
				goto glare
			label oops
			`"Justice," he says as he procures a hidden blaster pistol from his shirt. Before you can realize your error, he suddenly shoots you several times in the stomach. You have just enough life in you to watch him be carted off by other Navy personnel before your vision is obscured by blackness.`

> 			`You are dead. Let this be a lesson to you - if you act like an asshole, the galaxy tends to respond in kind.`

Reviewer 2: Unless you happen to be Hines.

The "Lesson" seems really heavy handed.  And considering the player is... several missions post-Hadley, loading an earlier save may not be available to "avoid the bad-end".

And why isn't there any check for "helped Hadley"?

			`	His expression suddenly softens. "The name's Henry Radcliff. I need to talk to you in private, if you don't mind." You nod, then turn to the other auxiliary captains. "I'll handle this one," you say to them.`
			`Back aboard your ship, you lock the doors to your quarters before you and Radcliff sit down. "I'm glad I found you, Lieutenant," he begins. "Ever since we both started working for the Syndicate, Lucas and I made a promise to watch each others' backs. And it's my understanding that you made a promise to him, as well?"`
			`	You nod. "When I learned that one of our officers had threatened Hadley, I wasn't about to let something like that slide."`
			`	"And I thank you for that - there's no way I could bring that to my boss, and anyone else would just ignore me. But with you here, I think we have a real chance at getting Hadley and his family to safety."`
			`	"So what's the plan?"`
			`Henry procures a small holo-projector. "When the bombings happened, Lucas and I knew we had to come up with a contingency in case things got ugly. After considering our options, we decided to select this place." The projector highlights Hermes in the Capella System.`

> 			`	"Hermes?" you say, with a sly grin. "That makes perfect sense. The planet's got orbital bombardment stations trained on the planet around the clock - if someone tried to tail you there, it's almost certain they'd be atomized by the relays."`

Reviewer 2: ... Yes, because the orbital bombardment system just fires at anyone...

The point of the orbital bombardment system is to clean an area before anything infectious spreads.  It's not there for planetary defense.  Tailing someone there would be a bit obvious, but so would having a family living there.  It's research stations mostly, and people living there need a government permit.

> 			`	"That's exactly what we were thinking. And it's still close enough to Syndicate Space that it wouldn't be that far of a trip. Ideally, we wanted to settle in the Deep, but that's too far away."`

Reviewer 2: How is it too far away?  5~6 more jumps through pretty safe space, to land in the Deep where there's some pretty good protection?

			`The pair of you stand up. "I'll be sure to relay this to my superior officer, Henry," you reply, shaking his hand. "He's been trying to come up with a plan to get Hadley to safety, and this will be the break he needs."`
			`	"I want to thank you once again, Lieutenant...?"`
			`	"<last>. <first> <last>, if you want, Henry."`
			`	"Thank you, <first>. Really, I can't thank you enough."`

> 			`A thought enters your head. "Henry...I know this might be asking a lot, but if it's possible, I have a favor to ask. I'm sure you're well-aware of the bombings of Geminus and Martini at this point - if you get any leads, can I ask you to be my eyes and ears?"`

Reviewer 2: Why is the player now investigating the bombings?

Did I miss something where it's called into doubt as to who did them in this storyline?  As far as I know, the Republic/Navy assumes it's the FW who did it, until evidence is presented.

			`"You got it, <last>," he replies without hesitation. "After all, you're bending over backwards to make sure one of my best friends is safe and sound. I'll let my other buddies know to keep an eye out for you - the more feelers we can put out, the better."`
			`The pair of you soon exit your ship and head back to the auxiliary captains. "I won't let you down, Lieutenant," Henry says as he gets back in line. "Thanks for giving me this chance!"`
			label end
			`It takes a couple hours before all the mechanics have loaded onto the auxiliaries. With your errand complete, you board the <ship> and get ready to head back to Geminus.

	on complete
		payment 2500000
		dialog
			`As you break through the clouds over Geminus, you are treated to the awe-inspiring sight of the new shipyard unfold before you. "Welcome to the new Geminus, <last>," Danforth congratulates you. "This represents the first step of many on the road to victory. To celebrate, take these 2.5 million credits - I'm sure there's something in the shipyard you might want."`

> event "geminus rebuilt"

Reviewer 2: So the entire shipyard/facilities are rebuilt in... days? a month or two?
I know the player can take their time with some missions, but the shortest time would be ... pretty amazing for rebuilding a whole Navy shipyard.  FW forces you to take what... 2x30 day breaks, plus the whole campaign and 90 days after, for Geminus to be rebuilt.

		spaceport `Like everything else on Geminus, the spaceport is not terribly pretty, but is functional and efficient. Each of the wide hallways is marked out like a road, with space in the center for electric carts carrying cargo, while pedestrians are relegated to the edges. Video cameras and sensors have been placed at every intersection, and access to the shipbuilding facilities is tightly controlled.`
		spaceport `	The hub of the spaceport is a food court with a ring of shops in the center and seating on the outside, where you can look out of plate glass windows at the lava fields and caldera outside. By night, you can see the volcanic glow all around the horizon.`
		shipyard "Basic Ships"
		shipyard "Navy Basics"
		shipyard "Navy Advanced"
		shipyard "Betelgeuse Basics"
		shipyard "Lionheart Basics"
		outfitter "Common Outfits"
		outfitter "Ammo North"
		outfitter "Lovelace Advanced"
		outfitter "Syndicate Basics"

mission "Tier Boost"
	landing
	source
		government "Republic"
	to offer
		has "Geminus 3C: done"
		has event "geminus rebuilt"
	on offer
		log "Was promoted because of my assistance with rebuilding the shipyards on Geminus."
		"reputation: Republic" += 10

> 		"reputation: Free Worlds" = 500

Reviewer 2: I assume this is just because there's that rep-reset at the end of FW, right?
Otherwise, there's no reason to be friendly with them, seeing as you're at war with them.

# Copyright (c) 2020 by Bereskatuket
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# Republic/Navy Campaign Project Lead: Bereskatuket
# Head Writer/Creative Consultant/Resident Grammar Nazi: Warlord Mike
# Opening Missions Writing Credit to: Disiuze, Kiko, Moonbeam, Kat
# 

mission "Republic Bounty 1"
	name "Republic Bounty Hunting"
	description `Hunt down the <npc>, a pirate ship that has been assaulting miners in the Boral system.`
	minor
	source
		government "Republic"
		attributes "dirt belt"
	to offer
		has "event: war begins"
		"combat rating" > 40
		not "chosen sides"
		random < 65

	on offer
		conversation

> 			`You barely have time to find your bearings in this spaceport when you're approached by a man in a crisp, navy blue Republic uniform.`

Reviewer 3: "Navy blue" is a color: https://en.wikipedia.org/wiki/Navy_blue

This seems fine to me.

			`	"Your file led me to believe otherwise," says Shah. "Either way, we will be sure to update our record and will be working with other contractors in the future."`
				decline
			label asking
			`	Her face darkens. "I won't try to convince you if you're not interested."`
			choice
				`	"Okay, I'll do it."`
					accept
				`	"Thank you, but I'm not interested in this kind of work."`
					decline
	npc kill
		personality heroic staying nemesis target
		government Bounty
		system
			near "Alnasl" 1 2
		ship "Bastion (Heavy)" "Amphithere"
		dialog `The <npc> has been eliminated. You can claim the bounty payment by returning to <destination>.`

	on accept
		log `Hunting a pirate ship called the "<npc>," which is wanted by the Republic. It was last spotted in the Alnasl system.`

	on complete
		payment 200000
		dialog `Captain Shah contacts you after you've landed. "The Republic thanks you for your assistance. <payment> have been credited to your account. I'll be in touch should I have need of you again."`
		log "People" "Anton Kent" "Anton Kent is a Republic official assigned to keep an eye on the systems bordering the Free Worlds. He seems to be unaffiliated with the Navy."

> 		log "People" "Amelia Shah" "Amelia Shah is a Republic official assigned to keep an eye on the systems bordering the Free Worlds. She seems to be unaffiliated with the Navy."

Reviewer 4: Lack of uniform probably.

	npc disable
		personality heroic staying nemesis target
		government Bounty
		system "Tais"
		ship "Falcon (Heavy)" "Gediminas"
		dialog `The <npc> has been disabled. You can claim the bounty payment by returning to <destination>.`

	on accept
		log `Hunting a pirate ship called the "<npc>," which has captured multiple merchant captains. The ship was last sighted in the Tais system.`

	on complete
		"assisted republic" ++
		payment 250000
		dialog `Soon after you land, Captain Shah contacts you. "The hostages were recovered safely," she says, with a smile. "The Republic owes you a debt of gratitude. If you are interested in more tangible benefits, <payment> have been transferred to your account, and I have informed my superiors that you can be counted on."`
		log "Captain Shah says she has informed her superiors of my contributions."

mission "Republic Scouting 1"
	name "Hyperspace Relay Testing"
	description "Fly through the <waypoints> systems, then report to <destination> so the Navy can analyze the network connectivity of your logs."
	minor
	waypoint "Hassaleh"
	waypoint "Sumar"
	waypoint "Cardax"
	waypoint "Volax"
	source
		government "Republic"
		attributes "paradise"

> 	destination "Glaze"

Reviewer 5: ```suggestion
	destination "Farpoint"
```

Reviewer 5: Change destination from a Free Worlds planet, to the north side of the mission waypoints

Reviewer 4: I suspect Glaze was the original choice simply because when this was first raised a while back the concern was made that most new players would end up playing the republic campaign 'by default' simply because they were there, so there was an intention to force people to see other factions before they got locked into anything.
That may not have translated well into actual missions though.

Author: Yeah. I don't know why it was chosen to be Glaze. Farpoint makes more sense.

		payment 50000
		log "Captain Melgrew says she has put a good word in for me with her superiors."

mission "Republic Shipping 1"
	name "Secret cargo to <destination>."
	description "Transport a squad of Navy marines and their undisclosed cargo to <destination>."
	minor
	passengers 5
	cargo "undisclosed cargo" 5
	to offer
	source
		government "Republic"
		attributes "factory"
	to offer
		has "event: war begins"
		not "chosen sides"
		random < 50
	destination "New Austria"

	on offer
		conversation
			`Shortly after leaving your ship, you notice a squad of what appears to be Navy Marines gathered around a civilian transport. The ship looks damaged, and the smell of hot metal fills the air.`
			choice
				`	(Talk to them.)`
				`	(Ignore them.)`
					decline
			`	You approach who you assume to be the captain of the ship and ask them what happened.`
			`	"I was hired to take these fine individuals and their cargo to <destination>," the captain replies. "Unfortunately, I don't think I can make it the rest of the way. Would you be willing to take over? Oh, and don't bother asking what they are transporting, they won't tell me."`
			choice
				`	"I can do that."`

> 				`	I'd prefer knowing what it's in my hold, sorry."`

Reviewer 5: ```suggestion
				`	"I'd prefer knowing what it's in my hold, sorry."`
```

Reviewer 5: Missing quote

			choice
				`	I'm interested - I'll take on that ship.`
					goto jobAccept
				`	Sorry, but I have no desire to join the navy at this time.`
					defer
                    
			label jobAccept
			`	[We look forward to seeing you on <destination> once the mission is over,] the message reads. [Good luck!]`
					accept

	on accept
		log "Accepted an enlistment offer from the Republic Navy."
            
	npc kill
		personality heroic staying nemesis target
		government "Pirate"
		system "Castor"
		ship "Raven" "Blackfury"
				
	on complete
		"reputation: Republic" += 10
		"reputation: Free Worlds" = 500
		"reputation: Escort" = 500
		"salary: Navy" = 500
		set "license: Navy"
		event "naval employment earned"
		event "Navy Painted"

> 		log "Was able to destroy the pirate ship and prove my worth to the Navy. The Navy has inducted me as a Private and given me a small salary."

Author: Missed that one. Will update.

			`	"We look forward to seeing you on <destination> once the mission is over," the message reads. "Good luck!"`
					accept

	on accept
		log "Accepted an enlistment offer from the Republic Navy."
            
	npc kill
		personality heroic staying nemesis target

> 		government "Pirate"

Reviewer 5: ```suggestion
		government "Bounty"
```
This ship just gets obliterated out of the sky with all the Navy capital ships around!

> 			`Howell's expression hardens as his gaze turns left and skyward. "With rumblings of war starting in the South, and pirates massing in the North, the Navy is going to be stretched thin." His gaze returns to you. "As of today, you haved been approved for Navy Tier I Enlistment status. As part of being inducted as a Navy Private, you'll receive a daily stipend of 500 credits per day. At ease."`

Author: And this one, too.

			choice
				`	"Understood, sir. I won't let you down."`
					goto missionAccept
				`	"Sorry, sir, but I don't know if I'm ready just yet."`
					goto missionDecline

			label missionAccept
			`	"Very good. Now, remember - this is a scouting mission. I do not expect, much less require, that you destroy every pirate ship you find. Engage them if you feel you must, but if it becomes clear you're in over your head, jump to hyperspace. Make it back here once your patrol is complete, and there's 100,000 credits waiting for you, in addition to whatever financial gratitude other Navy ships will lend."`
					accept

			label missionDecline
			`	"Don't worry, Ensign - there's no real timetable for the scouting, but I understand your hesitancy. Contact me again when you think you're ready."`
					defer
	
	on complete
		payment 100000
		dialog
			`"Good job making it back in one piece," Danforth congratulates as a lower-ranking intelligence officer quickly collects your ship's sensor logs while handing you your payment. "Make your way to the spaceport when you're ready for your next assignment."`

mission "Navy Pirate Patrol 2"

> 	landing

Reviewer 5: ```suggestion
```
Previous mission says to meet Danforth in the spaceport.

			`Most of the officers quietly talk amongst themselves at your response, but Hines keeps staring you down. You don't know why, but it seems like he feels threatened by you speaking as though you were a higher rank than he was.`
			`	"Ensign <last>, at ease," Danforth finally says as he stands up from behind his desk. "We can go over the details later in the spaceport," he adds, walking towards you. "In the meantime, here's your payment of 250,000 credits, as promised."`

mission "Navy vs Pirate"
	landing
	name "Destroy <npc>"
	description "A pirate vessel named <npc> has broken through the Navy defenses in Alnitak and is currently in the Saiph system. Destroy it before it can move further into Republic space and return to <planet> for additional orders."
	to offer
		has "Navy Pirate Patrol 3: done"
	source "Farpoint"
	waypoint "Saiph"
	on fail
		dialog `You have failed an essential Navy mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`

> 	npc kill

Reviewer 5: NPC name is needed here, ends up with a blank in the mission conversation.

		conversation

> 			`Farpoint feels a lot different when the entire base is on alert. Normally, you could easily see from one side of the compound to the other and only have the odd soldier or vehicle break your vision. Since the pirate ship that broke through, however, the farthest you can see is maybe 50 feet due to the mass of guards, crewmen, machinery and munitions.`

Reviewer 4: yeah, no imperial measurements.

			`Danforth scoffs, smiling. "Don't ever be afraid to ask, Ensign - honestly, you being here helps me take my mind off of things."
			label mission
			`	Danforth procures a small holo-projector from his pocket, which shows the nearby systems. "We've managed to track the pirate to the Saiph system, <first>," he says, right as Saiph is highlighted. "The ship in question is a Leviathan-class warship known as the '<npc>' - they've been reported terrorizing merchants passing through the system. Once you destroy the <npc>, report back here for further instructions."`
			choice
				`	"Consider it done, Admiral."`
			`	"Be careful, <last> - while I am able to tell you it's a Leviathan, our teams were unable to successfully get a good read on its weapon systems. Be ready for anything."`
				accept
	on accept
		log "Have been assigned to take out a pirate that broke through the Navy defenses and is ambushing merchants in Saiph."
	on complete
		log "Succeeded in destroying the pirate ship in Saiph. For my efforts, I have been promoted to the rank of Lieutenant, with an impending increase to my salary."
		payment 350000
		conversation

> 			`As your ship pierces <destination>'s atmosphere, you notice that the base has reverted to its much-less active state. It isn't long before the control tower radios in: [Ensign <last>, proceed to Hangar 357 - Admiral Danforth is expecting you.]`

Reviewer 6: ```suggestion
			`As your ship pierces <destination>'s atmosphere, you notice that the base has reverted to its much-less active state. It isn't long before the control tower radios in. "Ensign <last>, proceed to Hangar 357 - Admiral Danforth is expecting you."`
```

				goto close
			label equipment
			`	He looks awfully pleased with himself. "Yes, she is quite the beauty. I would have never been able to afford anything like this without the Defense Pact's scouting program," he says. "She makes keeping pirates out of our territory as easy as a dream. If you're interested in trading that tub you've got in for something like this, the program should be hiring again soon."`
			label close
			`	You cut the comlink and breathe a sigh of relief that the conversation is over.`

	on complete
		"assisted republic" ++
		dialog `Soon after you land, Jane comes to your ship and pays you <payment> in exchange for your sensor logs regarding <npc>. "No alien tech, I see," she says, sounding dissapointed. "While that would have been a great find, I will admit I'm happy I don't have to figure out how to defend against it. I've let my supervisors know how helpful you've been, hopefully we can work together again soon." `
		payment 50000
		log "Captain Melgrew says she has put a good word in for me with her superiors."

mission "Republic Shipping 1"

> 	name "Secret cargo to <destination>."

Reviewer 5: ```suggestion
	name "Secret cargo to <planet>"
```

Reviewer 5: Use <planet> instead of <destination> for mission titles, otherwise they'll almost always overflow.

![image](https://user-images.githubusercontent.com/4471575/95669454-88764280-0b46-11eb-9b50-90da26e4676b.png)

			`	"I was hired to take these fine individuals and their cargo to <destination>," the captain replies. "Unfortunately, I don't think I can make it the rest of the way. Would you be willing to take over? Oh, and don't bother asking what they are transporting, they won't tell me."`
			choice
				`	"I can do that."`
				`	"I'd prefer knowing what it's in my hold, sorry."`
					decline
			`	"Great! The name's Jeff Haas, by the way. I run a shipping company mostly taking contracts for the Navy," Jeff says enthusiastically.`
			`	"Complete this job and I will gladly hire you to do more in the future."`
				accept

	on accept
		log "Accepted a job transporting some marines and their cargo to <destination>."

	on complete
		payment
		payment 50000
		log "People" "Jeff Haas" "Jeff Haas is the head of the Jeff Haas Shipping Company. He is often contracted by the Republic Navy."
		dialog `After a very quiet journey, you land on <destination>. The marines unload their cargo without saying a single word to you. A different navy officer thanks you for your assistance, and pays you the respectable sum of 50,000 credits.`

mission "Republic Shipping 2"

> 	name "Navy Interior Design to <destination>"

Reviewer 5: ```suggestion
	name "Navy Interior Design to <planet>"
```

	description "Transport furniture to the Navy offices on <destination>."
	cargo "furniture" 20
	to offer
	source
		government "Republic"
		attributes "factory"
	to offer
		has "event: war begins"
		has "Republic Shipping 1: done"
		not "chosen sides"
		random < 50
	destination "New Austria"

	npc accompany save
		personality escort timid
		government "Merchant"
		ship "Behemoth (Heavy)" "Alba Oro"

	on offer
		conversation
			`As you wander around the spaceport, a merchant captain calls you over and asks you to help transport some furniture to the Navy base on <destination>. You can clearly see the Jeff Haas Shipping logo on the cargo ramp of her ship.`
			choice
				`	"Of course."`
					accept
				`	"I don't have space for this right now."`
					defer
				`	"I'm not interested, and please don't offer me any more jobs in the future."`
					decline

	on accept
		log "Accepted a job to transport some furniture to <destination>."

	on complete
		payment
		payment 30000
		dialog `When you land, the merchant captain thanks you for the escort and use of your cargo space. She gives you a cut of the total payment, <payment>.`

mission "Republic Shipping 3"

> 	name "Convoy to <destination>"

Reviewer 5: ```suggestion
	name "Convoy to <planet>"
```

					accept

	on accept
		log "Accepted an enlistment offer from the Republic Navy."
            
	npc kill
		personality heroic staying nemesis target
		government "Pirate"
		system "Castor"
		ship "Raven" "Blackfury"
				
	on complete
		"reputation: Republic" += 10
		"reputation: Free Worlds" = 500
		"reputation: Escort" = 500
		"salary: Navy" = 500
		set "license: Navy"
		event "naval employment earned"
		event "Navy Painted"
		log "Was able to destroy the pirate ship and prove my worth to the Navy. The Navy has inducted me as a Private and given me a small salary."
		conversation
			`As you cycle the hatch to exit the ship, you are greeted by two Navy personnel flanking the entryway, armed with rifles. Between them, a Navy officer stands with his arms folded in front of him.`
			``
			`	"I assume you are <first> <last>?" he asks. You nod, saluting with your right hand. "Yes sir. And you are...?"`
			`	"Colonel Marcus Howell," he replies. "Well met, Captain <last> - or rather, Ensign <last>."`
			`	You nod, still saluting. "Thank you, sir. I hope to be of great service to the Republic."`

> 			`	"I have a feeling you'll be just the man we need, <last>," he states while making a slight bounce on the tips of his feet, smiling. "The speed and skill with which your ship, the <ship>, destroyed that Raven, in addition to your past endeavors, proves you are both serious about joining the Navy...and it's ready for battle."`

Reviewer 5: ```suggestion
			`	"I have a feeling you'll be just the pilot we need, <last>," he states while making a slight bounce on the tips of his feet, smiling. "The speed and skill with which your ship, the <ship>, destroyed that Raven, in addition to your past endeavors, proves you are both serious about joining the Navy...and it's ready for battle."`
```

Author: Whoopsie!

Darn it, all the times I tried to keep <first> <last>'s gender ambiguous and I completely spaced on that one.

		attributes "dirt belt"
	to offer
		has "event: war begins"
		"combat rating" > 40
		not "chosen sides"
		random < 65

	on offer
		conversation
			`You barely have time to find your bearings in this spaceport when you're approached by a man in a crisp, navy blue Republic uniform.`
			`	"You have a face that says you're looking for work," he says, in a surprisingly cheerful tone. "We're in need of someone to deal some pirates that have been causing trouble near Boral. Not normally something we'd contract out, but, luckily for you, we're a little short on free ships right now."`
			choice
				`	"Sure."`
				`	"Sorry, I'm not really into bounty hunting."`
					defer
			`	"Much obliged," he says. "It's a ship called the <npc>. It's a small little devil, but don't let it take you off guard."`
				accept
	on accept
		log `Hunting a pirate ship called the "<npc>," which has been plundering ships near Boral.`

	npc kill
		personality heroic staying nemesis target
		government Bounty
		system
			near Boral 1 2
		ship "Clipper (Heavy)" "Black Fang"
		dialog `The <npc> has been eliminated. You can claim the bounty payment by returning to <destination>.`

	on complete
		payment 175000

> 		dialog `Upon landing, the commander greets you, seemingly surprised by your return. "The Republic needs more captains like you. Hopefully this payment of <payment> will be enough to convince you to help us out the next time." You nod. "Captain <last>, correct?" he says, and barely gives you enough time to agree before he continues speaking. "Right. I'm Commander Kent, in case I didn't introduce myself. I'll be keeping an eye out for you in case we ever need more assistance." `

Reviewer 5: It can reflect that, in a broad way, the Navy is getting a little desperate. Hiring hiring merchants with little history certainly backs that up, and while the Navy usually has a lot of decorum, for some personnel, that could slip a bit. Although it probably should either be a little more exaggerated, or cut down to be more professional, yeah.

mission "Republic Bounty 2"
	name "Republic Bounty Hunting"
	description `Kill a known criminal aboard the <npc>, which was last spotted in the Alnasl system.`
	source
		government "Republic"
		attributes "dirt belt"
	to offer
		has "Republic Bounty 1: done"
		not "chosen sides"
		random < 70

	on offer
		conversation

> 			`Before you can decide where you want to go in the spaceport, you are flagged down by a Republic officer. This time, they're a dour woman with black hair in a tight bun. "Captain <last>?" she asks. You nod. "I'm Captain Shah. Commander Kent has you listed as a privateer." she says in a quick, clipped tone. "We've been made aware that a criminal has been spotted near Alnasl. He is wanted dead."`

Reviewer 4: If this is on a different world it's more likely that the system simply flagged an ID, she may not have known or needed to check who listed you.
```suggestion
			`Before you can decide where you want to go in the spaceport, you are flagged down by a Republic officer. This time, they're a dour woman with black hair in a tight bun. "Captain <last>?" she asks.`
			choice
						`			"That's me."`
			`"Good, good. I'm Captain Shah. I've been keeping an eye out for someone to deal with a little problem for us." she says in a quick, clipped tone. "We've been made aware that a notable criminal has been spotted near Alnasl and our system flagged you for being listed as a privateer. He is wanted dead."`
```

		log "Captain Shah says she has informed her superiors of my contributions."

mission "Republic Scouting 1"
	name "Hyperspace Relay Testing"
	description "Fly through the <waypoints> systems, then report to <destination> so the Navy can analyze the network connectivity of your logs."
	minor
	waypoint "Hassaleh"
	waypoint "Sumar"
	waypoint "Cardax"
	waypoint "Volax"
	source
		government "Republic"
		attributes "paradise"
	destination "Glaze"
	to offer
		has "event: war begins"
		not "chosen sides"
		random < 60

	on offer
		conversation
			`As you wander the spaceport you overhear a woman in a Navy uniform telling a bored-looking merchant about satellite dishes in the empty systems to the north.`
			`	"That sounds interesting." The merchant tells her, "But I've got a cargo bay full of medical supplies headed the other direction."`
			`	"Also an important mission," she replies. "I wish you safe travels."`
			choice
				`	(Ask her what's going on.)`
				`	(Walk away.)`
					decline
			`	"Do you need help with retrieving a satellite?" you ask the Navy officer, interested, as the other merchant captain walks away.`

> 			`	"Not quite." She replies brightly. "We've been experiencing some latency between Farpoint and Maelstrom. I'd need someone to fly through, then let me look at their logs so I can figure out what the trouble is."`

Reviewer 4: Canonically there are no hyperlane communication links through this part of space.

Reviewer 7: In FW Reconciliation you get this
https://github.com/endless-sky/endless-sky/blob/c1afeac04f64757780dce6c0a18ecfe5d7777534/data/human/free%20worlds%20reconciliation.txt#L1057
> "all communications links with Syndicate space have been cut. The Net relies on hyperspace links to transmit messages, and there are no transmitter relays in the uninhabited systems that now form the only remaining link to Syndicate space."
The whole of human space is linked, but not through those systems.

Reviewer: I was pointed to this reply, and asked to give a little feedback on it. 

> In FW Reconciliation you get this
> https://github.com/endless-sky/endless-sky/blob/c1afeac04f64757780dce6c0a18ecfe5d7777534/data/human/free%20worlds%20reconciliation.txt#L1057
> 
> The whole of human space is linked, but not through those systems.

All of human space is linked except when the plot demands it isn't. There is no reason why Maelstrom and Farpoint would not have some form of a hyperspace relay network between the two, especially as it is heavily implied that Farpoint was made into such a fortress world due to first contact with the Hai (Yet Maelstrom isnt for some reason?) So of course there would likely be a redundant relay for managing some sensor equipment, and keeping in loose contact with human patrols around hai space. 

This is something the Moon and I took into great consideration during the planning of these missions. We decided on the more likely story that the relays in these systems were destroyed or otherwise jammed by the Pug. We also considered that perhaps Moktar was the break point. In the end, we decided that the actual details didn't matter so long as they didnt create a massive plot hole, which neither of these solutions would. In both events, Syndicate space would still be cut off no matter which way it was spun.

		log "People" "Jane Melgrew" "Jane Melgrew is a Navy captain charged with all manner of odd jobs, from diagnosing hyperspace network problems, to scouting uninhabited systems."

mission "Republic Scouting 3"
	name "Navy Reconnaissance"
	description "Perform an outfit scan on the <npc> in Yed Prior, then report to Captain Melgrew on <destination>."
	source
		government "Republic"
	destination "Follower"
	to offer
		has "Republic Scouting 2: done"
		not "chosen sides"
		random < 40

	on offer
		conversation
			`As you step off your ship, you are greeted by an out-of-breath Jane Melgrew. "Captain <last>!" she exclaims. "Glad I caught you. I have another job, and I couldn't just send you a message for this one.`
			choice
				`	"What's going on?"`
				`	"Sorry, I'm really busy."`
					decline
			`	"We have reason to believe there is a ship loaded with alien tech somewhere in the south," she says. "As I'm sure you've heard, the worlds in this area have been organizing a rebellion. So far they've been peaceful, but if they are arming themselves with alien technology, that could be a major problem for us. Would you be able to track down the ship and scan it?"`

> 			choice

Reviewer 4: I will second Anarchist, this has enormous potential to be a much more fleshed out conversation.

		dialog
			`"Good job making it back in one piece," Danforth congratulates as a lower-ranking intelligence officer quickly collects your ship's sensor logs while handing you your payment. "Make your way to the spaceport when you're ready for your next assignment."`

mission "Navy Pirate Patrol 2"
	priority
	landing
	name "Pirate Patrol"
	description "Fly through Almaaz and Unagi, then deliver you sensor logs to Farpoint."
	waypoint "Almaaz"
	waypoint "Unagi"
	source "Farpoint"
	to offer
		has "Navy Pirate Patrol 1: done"
		not "chosen sides"
	on offer
		conversation
			`	A short stroll through the spaceport makes you realize Farpoint is appropriately-named - beyond the walls of the base, the scorching desert sands stretch farther than the eye can see. If this isn't the middle of nowhere, you could certainly see it from here. Which, you muse to yourself, isn't necessarily a bad thing - the remoteness and elevated security means someone would need unfathomably poor judgement to try anything out here.`
			`	You sit down on a bench near one of the pubs while you wait for the Admiral. Not 10 minutes after you arrive, you see Danforth walking towards you and you wave him over.`

> 			`	"Thank you once again for your assistance, Ensign <last>," he begins as he sits down next to you. "The sensor logs you provided don't seem to indicate much change in pirate activity - granted, those are just the three nearest systems, so it's somewhat expected." He sits up a bit straighter, leaning back slightly while doing so. "But now, we need to do a bit more in-depth analysis. The next systems we need scanned are Almaaz and Unagi - as you'll be passing by Freedom, which is well-known for its anarchy, this mission is beyond what most other merchants would be comfortable with...and as a result, the payment will also be substantially more. Are you up to it?"`

Reviewer 4: If you're already an ensign you're not a merchant anymore...

			`	You quickly disembark the <ship> to find Danforth's aide waiting for you. The two of you quickly hurry to the Admiral's quarters, and find he's joined by other Navy officers. As you enter the room, instictively saluting as you do so, all eyes shift away from the table and towards you - and the looks you receive, in particular Danforth's, make you feel uneasy.`
			`	"Ensign <last>, your timing is impeccable," Danforth says. "We just obtained your sensor logs, and it may have just been enough to prevent things from getting worse."`
			`	"...Worse, sir?" you ask, cautiously.`
			`	"Indeed," one of the officers says. "Shit well and truly hit the fan right as you got here."`

> 			`	"Hines, please," Danforth exasperatedly shushes the officer, raising his hand. "Just as we finished downloading your logs, a pirate fleet suddenly warped in. Most of them were either destroyed or routed back into the North, but one ship managed to escape. Most of us believe it's thanks to your sensor logs that we were able to halt their advance as much as we did."`

Reviewer 4: I think 'jumped' is the preferred terminology over 'warped'.

			choice
				`	(Ask him what he's looking at.)`
					goto ask
				`	(Wait for him to finish reading.)`
					goto wait

			label ask
			apply	
				set "ranulph's background"
			`	"Something wrong, Admiral?" you inquire, concerned.`
			`	"Oh, it's Hines," he dismissively comments as he looks up from his clipboard. "I'm just reading some performance commentary from his peers."`
			`	Your face twists into another uneasy expression. "I'm guessing that falls under the category of 'not good.'"`
			`	"Ranulph Hines always been a handful," the admiral replies, setting the clipboard down. "The man seems to have it in his head that by climbing the ranks quickly, he'll be seen as some sort of hero. While I admire his efforts to do his best, his means of getting there have been..." Danforth makes an aside glance, as if checking to see if Ranulph is in earshot. "Well, to say that it's underhanded would be a slight understatement."`
			choice
				`	"If you don't mind me asking, Admiral, exactly how bad are we talking?"`
					goto ranulph
				`	"I see."`
					goto pirateHunt

			label ranulph
			apply
				set "excessive force"

> 			`Danforth sighs sadly as he shakes his head. "I'll be honest with you, <first> - it wasn't as bad when he first joined, but recently, some of the things Ranulph's done are borderline terroristic. Still fully legal within the bounds of Galactic Law, but extremely questionable from a moral standpoint."`

Reviewer 4: I might've missed something earlier, but discussing this sort of thing seems like gross misconduct on Danforth's part unless you've established a close, trusting relationship. Even then it should probably come with a verbalised disclaimer.

			`	Upon reaching the docks, several Syndicate dockworkers call you over to a large collection of cargo containers filled with the building materials bound for Geminus. "Took you long enough," one says with a cigarette hanging out the side of his mouth. "We've been waiting here for five hours with our thumbs up our collective asses for you Navy jarheads. One of your own came by here about a month ago and acted like they owned the place - I tell ya, my boss doesn't pay me enough to put up with shit like theirs."`
			choice
				`	"Sorry to have kept you waiting. We came as quickly as we could."`
					goto end
				`	"You better keep that tongue of yours in check, my man, or you'll be choking on that cigarette."`
					goto fight
				`	"Hold on. 'One of our own?' You mean someone from the Navy?"`
					goto investigate
			label fight
			apply
				set "bad look"

> 			`The dockworker chuckles for a second, takes his cigarette out of his mouth and flicks it at you. You almost throw a punch before one of the captains of the auxiliaries steps between the two of you. "Sir, it's been a long day for all of us. Could you please help us load our cargo onto our ships?"`

Reviewer 4: This is a strong action on <first>'s part. Even if you're going to get stopped, the decision to want to throw a punch probably shouldn't be narration.

			`"Yeah, yeah, we'll get on it," he replies. "And thanks for proving my point, you jumpy little shit," he pointedly directs at you. Only then do you realize Danforth will probably hear of your behavior - and it's a sure bet he will not be pleased by your conduct.
				goto end
			label investigate

> 			`"Who the FUCK did you think I meant, asshole?!" he almost screams at you as the cigarette falls out of his mouth. "You know goddamn well I meant someone from the Navy! I knew you fuckers were uptight, but don't tell me you're all a bunch of fucking idiots, too!"`

Reviewer 4: This is a rather extreme reaction for a dockworker, I doubt most of these people are too seriously put-out by getting paid to *not* do work for a few hours of waiting.

			`	"The Navy is only responsible for protecting the Republic - the Syndicate tends to look after itself," you comment.`
			`	"See, that's what *I* told him. But the moment I did, he punched me right in the jaw and knocked me to the ground, telling me, 'you need to learn your place, you piece of shit.' Once we had loaded the materials into his ships, he pulled me aside and specifically told me, 'if you tell anyone of what I did, I will kill you, your family, and everyone you loved - and I have files on hand to help me.' As large as the Navy is, I was inclined to believe him."`
			`	"I'm guessing you didn't report this to your bosses."`
			`	"Hell, no. One, they wouldn't have done anything - most likely just brushed it off as 'he was having a bad day,' but that's the Syndicate for you. Two, they probably would've docked me a few days' pay for raising the issue - the Syndicate pays well, to be sure, but it's still just barely enough to make ends meet. And I wasn't about to call his bluff - no way am I putting my wife, my daughter and two sons in danger."`
			`	"Can you describe him?"`

> 			`	"Do you one better," he says as he stands up, walking over to a file cabinet. The drawers rattle and clunk as he pulls out a manilla-colored file with some photos inside. "I looked through security footage on the day he came through, in case I needed to 'disappear' for a bit." On a photo showing several Navy personnel walking alongside some forklifts, he points to one in particular - a face you quickly identify as Ranulph Hines. "That's him," he says, tapping Ranulph's face.`

Reviewer 4: Why does a dockworker have a file cabinet, and not just like, a photo on his phone or something?

			branch terror
				has "rivalry"
			choice
				`	"I see. I'll keep an eye out for him."`
					goto end
				`	"That looks like Ranulph Hines."
			`"Know him, eh?" he inquires. "I'd watch myself if I were you - man's got a mean streak a mile wide."`
					goto end
			label terror
			apply
				set "lifeDebt"
				log "People" "Ranulph Hines" `Hines seems to believe that the Syndicate is subservient to the Republic - Lucas Hadley, a Syndicate dockworker, was injured by Hines when Hadley pointed out the Syndicate was more than capable of handling itself, threatening Hadley and his family with death if he reported Ranulph's behavior.`
				log "People" "Lucas Hadley" `A Syndicate dockworker from Lodestone, Lucas Hadley was attacked by Ranulph Hines for challenging Hines' belief that the Republic are the superior force in human space, when the Syndicate is quite capable of defending itself from outside threats. Hadley puts on a gruff exterior (and a colorful tongue, to match), but deep-down, he's a good man who does his best to provide for his family.`
			choice
				`	"I had a feeling it might be Ranulph Hines."`
			`"Really? What makes you say that?" he asks, slightly alarmed.`

> 			`	You quickly explain what Danforth told you - how Ranulph feels the need to make himself seen as a "hero of the Republic," but his actions prove he is anything *but.* With some trepidation, you even mention how Hines dealt with the slavers to the dockworkers, calling his actions "reprehensible," and can't believe a man such as he would even be allowed to serve in the Navy."`

Reviewer 4: This is definitely misconduct.

			`The admiral's quarters turns out to be a small grouping of construction trailers connected to each other. "Shut the door, please," he says as you enter the office. As Danforth takes a seat behind his desk, you glance about the room. The walls have tables, filing cabinets, and a few calendars with target objectives to be met.`
			`	"I understand there was...an incident...on Lodestone. Is that accurate?" Before you can even reply, he shows you a video recording from the docks where you almost got in a fight. Watching in wide-eyed horror as your actions are displayed for all to see, you feel the temperature in the room drop sharply.`
			`"In case you're wondering, the dockworkers sent me that recording."`
			`	"Sir, I-"`
			`	"Don't," he harshly cuts you off, standing up. "I don't need to hear your excuses, <last> - your conduct speaks volumes more than your words. And for every action...there are consequences." Suddenly, you receive an email from your bank account - 1 million credits were just returned to the Navy. In the brief moments you look at your communicator's display, Danforth is already uncomfortably close to you.`

> 			`	"Rather than revoke your salary, I opted for something a bit more...immediate," he explains, calmly. "As I gave you the 1.5 million, I can just as easily take it away."`

Reviewer 4: The ramifications of this for how the banking establishment in the republic works are frankly dystopic and terrifying.

			`	"That's the extent of your mission - get to <stopover> with the auxiliaries, pickup the troops and guns, and get them back here. Is there anything else you need to discuss before you leave?"`
			`	"Actually, sir...there is."`
			`Danforth raises an eyebrow. "Alright, I'm listening."`
			`	"I met a dockworker at Lodestone named Lucas Hadley. He says that Hines stopped by about a month ago to pick up some materials for the base at New Iceland - is that true?"`
			`"Let me see," Danforth says, checking his computer. It takes a moment, but he eventually finds his calendar. "Ah, yes. I had sent him out to pick up some framulators for the base. Though I'm curious, <first>...why do you ask?"`
			`You glance over your shoulder, even though there's no-one who could possibly see you. "Admiral, Hines apparently *decked* Hadley because the latter offended his twisted sense of honor."`

> 			`"What?!?" he exclaims, visibly shocked. "Lieutenant, this is a *very* serious accusation you're making," he warns, gravely. "I hope you have evidence to substantiate your claim."`

Reviewer 4: Is he loud or is he grave? This all on one line with no break is like getting whiplash.
